### PR TITLE
chore(config): Add Travis CI support

### PR DESCRIPTION
**Pull request by frassinier**
In order to generate some SASS documentation and start using gh-pages linked to this folder.
https://talend.github.io/bootstrap-theme/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  directories:
+    - node_modules
+notifications:
+  email: false
+branches:
+  only:
+    - master
+node_js:
+  - "6"
+  - "4"
```
Comment (jmfrancois) [inline .travis.yml]: I have just added 4 because it's LTS

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+after_success: |
+  if [ -n "$GITHUB_API_KEY" ]; then
+    cd "$TRAVIS_BUILD_DIR"
+    # This generates a `docs` directory containing sassdoc to publish.
+    cd docs
+    git init
+    git checkout -b gh-pages
+    git add .
+    git -c user.name='travis' -c user.email='travis' commit -m 'Generate gh-pages from CI'
+    git push -f -q https://frassinier:$GITHUB_API_KEY@github.com/Talend/bootstrap-theme gh-pages &> /dev/null
```
Comment (jmfrancois) [inline .travis.yml]: why is their your github login here ?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    cd "$TRAVIS_BUILD_DIR"
```
Comment (jmfrancois) [inline .travis.yml]: finish a script with a cd seems weird to me. Any reason for that ?

File: .travis.yml (hunk 1 above)
Comment (talend-glorieux) [inline .travis.yml]: Building the documentation on two different version of nodejs is not required.
No tests are ran during this build.

File: .travis.yml (hunk 2 above)
Comment (frassinier) [inline .travis.yml]: Since it's my Github API token, it's needed :(

File: .travis.yml (hunk 3 above)
Comment (frassinier) [inline .travis.yml]: I copy this shell from tutorial but I imagine that it's for another scripts triggered after `after_success`?

File: .travis.yml (hunk 2 above)
Comment (jmfrancois) [inline .travis.yml]: I don't know how we could handle that in a different way. Let's do it that way at the moment except if someone has an other idea.
May be @talend-glorieux you know that better than me.

File: .travis.yml (hunk 3 above)
Comment (jmfrancois) [inline .travis.yml]: oh I understand, It's like go back to previous state in the case of sth else should be done.

File: .travis.yml (hunk 2 above)
Comment (sgandon) [inline .travis.yml]: May be you can ask to Irene or someone from devops if they have an account for that, they must have an account for the jenkins server to get access to Github.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+branches:
+  only:
+    - master
+node_js:
+  - "6"
+  - "4"
+before_install:
+  - npm i sassdoc -g
+before_script:
+  - npm prune
```
Comment (acateland) [inline .travis.yml]: why pruning ?

Comment (frassinier) [inline .travis.yml]: Since I asked to set _node_modules/_ in cache

File: .travis.yml (hunk 3 above)
Comment (frassinier) [inline .travis.yml]: I think deal with this env var is more secure than relative paths

File: .travis.yml (hunk 4 above)
Comment (jmfrancois) [inline .travis.yml]: yes it's quite usual to add it in before_script when you speed up your tests with that cache

**Pull request merged**